api/v1alpha1/database_types.go

```diff
@@ -145,6 +145,10 @@ type DatabaseSpec struct {
 	// (Optional) If specified, the pod's priorityClassName.
 	// +optional
 	PriorityClassName string `json:"priorityClassName,omitempty"`
+
+	// (Optional) If specified, the pod's terminationGracePeriodSeconds.
+	// +optional
+	TerminationGracePeriodSeconds *int64 `json:"gracePeriod,omitempty"`
```

Comment:
it is better to use terminationGracePeriodSeconds like in PodSpec as we do with other PodSpec attributes here